hack/update-generated-swagger-spec.sh

```diff
@@ -49,7 +49,7 @@ generated_file="${SWAGGER_SPEC_OUT_DIR}/openshift-openapi-spec.json"
 oc get --raw "/swagger.json" --config="${MASTER_CONFIG_DIR}/admin.kubeconfig" > "${generated_file}"
 
 os::util::sed 's|https://127.0.0.1:38443|https://127.0.0.1:8443|g' "${generated_file}"
-os::util::sed -E 's|"version": "[^\"]+"|"version": "latest"|g' "${generated_file}"
+os::util::sed -E '0,/"version":/ s|"version": "[^\"]+"|"version": "latest"|g' "${generated_file}"
```

Comment:
Does the first replace in the file then leaves the rest of the instances alone.  "Between lines 0 and the line containing the first match, do the following replace".  `jq` would be better here, but...